docs/tracking.md

```diff
@@ -18,31 +17,26 @@ hence this page does not aim to track pure Python packages.
+| NumPy             | :material-check-bold: |                       |           2.1.0            |               :simple-linux: :simple-apple:                |    [:simple-anaconda:](https://anaconda.org/scientific-python-nightly-wheels/numpy/)     |
+| packaging         | :material-check-bold: | :material-check-bold: |            24.0            |                                                            |                                                                                          |
+| pandas            | :material-check-bold: |                       |           3.0.0            |                                                            |                                                                                          |
+| Pillow            | :material-check-bold: |                       |                            |                                                            |                                                                                          |
+| pip               | :material-check-bold: | :material-check-bold: |            24.1            |                                                            |                                                                                          |
+| pybind11          | :material-check-bold: | :material-check-bold: |            2.13            |                                                            |                                                                                          |
+| PyWavelets        | :material-check-bold: |                       |           1.7.0            |               :simple-linux: :simple-apple:                |  [:simple-anaconda:](https://anaconda.org/scientific-python-nightly-wheels/pywavelets/)  |
+| scikit-build-core | :material-check-bold: | :material-check-bold: |           0.9.5            |                                                            |                                                                                          |
+| scikit-learn      | :material-check-bold: |                       |           1.6.0            |                       :simple-linux:                       | [:simple-anaconda:](https://anaconda.org/scientific-python-nightly-wheels/scikit-learn/) |
+| SciPy             | :material-check-bold: |                       |           1.15.0           |               :simple-linux: :simple-apple:                |    [:simple-anaconda:](https://anaconda.org/scientific-python-nightly-wheels/scipy/)     |
+| setuptools        | :material-check-bold: | :material-check-bold: |           69.5.0           |                                                            |                                                                                          |
+
+[^cmake]: Windows isn't correctly handled yet in CMake 3.30.0, see [cmake#26016](https://gitlab.kitware.com/cmake/cmake/-/issues/26016)
```

Comment:
why did the table get reformatted?

Reply:
It's now all formatted and aligned in single lines, which I guess makes it a bit easier for a human to parse. At least it feels that way to me.

Before:
<img width="1430" alt="Screenshot 2024-07-19 at 15 30 23" src="https://github.com/user-attachments/assets/2c4ec240-5d52-40d1-af57-9ed5dd08da9e">

After:
<img width="1385" alt="Screenshot 2024-07-19 at 15 30 54" src="https://github.com/user-attachments/assets/f26172ca-cd74-4552-b0f3-965dadd1e872">

Reply:
Nice I agree this is better, especially if your editor is configured not to line wrap.

Reply:
FWIW, if you're using VS Code on macOS, Option+Z toggles between line wrap mode.